src/transform.ts

```diff
@@ -47,3 +52,21 @@ function dumpTree(xformed: ESTree.Program, url: URL) {
   writeFileSync(path, JSON.stringify(xformed, null, 2));
   treeDebug("wrote transformed tree to %s", path);
 }
+
+// HACK: In node 18/16, when using --loader node-ts/esm
+// sourcemap gets inserted twice to the typescript file. We remove the second one
```

Comment:
This is weird. Probably there's a better way to handle this (and somehow avoid double sourcemap, which suggests the source gets transpiled twice), but I suppose it's okay for now.